inspirehep/utils/url.py

```diff
@@ -101,3 +101,8 @@ def get_legacy_url_for_recid(recid):
     """
     pattern = current_app.config['LEGACY_RECORD_URL_PATTERN']
     return record_url_by_pattern(pattern, recid)
+
+
+def get_hep_url_for_recid(recid, endpoint):
+    inspirehep_url = current_app.config.get("INSPIREHEP_PUBLIC_URL")
```

Comment:
isn't it `INSPIREHEP_URL` enough?

Reply:
`INSPIREHEP_URL` is actually internal url (no domain, but ip address and it points to /api endpoint)

Reply:
oh I see, we could use `SERVER_NAME` then :)

Reply:
Yeah, but this is only domain without protocol. I can use it if you prefer but I will still have to build proper url.

Reply:
Additionally on tests it's `localhost:5000` so it will make tests less readable as it will point to `next` instance.

Reply:
```python
In [4]: app.config['PREFERRED_URL_SCHEME']
Out[4]: 'https'

In [5]: app.config['SERVER_NAME']
Out[5]: 'inspirehep.net'
```
We can combine them together instead of introducing yet another config :)

Reply:
Yes I know, this is what we are already doing on hep. Still on next as I said SERVER_NAME in tests is "localhost:5000" which means it's pointing to next instance not to hep instance. And logically this address should not point to NEXT but to HEP

Reply:
hmm what is the problem on tests if it's `localhost:5000`? You could even patch it or we just check if the url is correct with whatever is in `SERVER_NAME`

Reply:
Yeah, I also know this but this just feels wrong, as logically "SERVER_NAME" is server name for NEXT not fo HEP and the URL we want to generate in there is for HEP.

Reply:
Ok, I changed it but we have to remember that if we change SERVER_NAME it will change in curator tickets also.